src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml

```diff
@@ -130,7 +130,7 @@
                                       Style="{StaticResource InCanvasScrollViewerStyle}"
                                       CanContentScroll="True"
                                       Width="250">
-                            <ContentControl Template="{StaticResource MemberControlTemplate}" />
+                            <ContentControl Template="{StaticResource InCanvasMemberControlTemplate}" />
```

Comment:
Thanks @lillismith. Hi @aosyatnik, I have created this task for you to track internally:

[MAGN-7780](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7780) Remove full method signature from library UI

That can be done as a separate PR and get this merged in first (once you've confirmed the testing I mentioned below is good).